src/Athena/Content/css/site.css

```diff
@@ -206,4 +206,39 @@ main {
 
 .courseTable table {
     width: 100%;
+}
+
+.navChip {
+    background-color: lightgray;
+}
+
+    .navChip:hover {
+        color: white;
+        background-color: #2962ff;
+    }
+
+.tooltip {
+    position: relative;
+    display: inline-block;
+}
+
+    .tooltip .tooltiptext {
```

Comment:
So it looks like [materialize css tooltips](http://archives.materializecss.com/0.100.2/dialogs.html#tooltip) doesn't work inside a nav. I'd like to use it elsewhere so let's try to match their style as close as possible. Let's start by dropping the boarder rounding to `2px`, setting `top` to `110%` and removing the width and height limitations.